BSDmakefile

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright (C) CFEngine AS
+#  Copyright 2017 Northern.tech AS
```

Comment:
This is the first source code that I see that deletes copyright history and mentions copyright only for the current year like it just got created.